test/fixture/generatedTypeScript/interfaces/IType.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/prefer-default-export, import/no-cycle, @typescript-eslint/naming-convention, @typescript-eslint/no-empty-interface */
+export interface IType {
```

Comment:
Codacy found an issue: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)